spring-rabbit/src/main/java/org/springframework/amqp/rabbit/annotation/RabbitListener.java

```diff
@@ -89,11 +90,13 @@
  * @since 1.4
  * @see EnableRabbit
  * @see RabbitListenerAnnotationBeanPostProcessor
+ * @see RabbitListeners
  */
 @Target({ ElementType.TYPE, ElementType.METHOD, ElementType.ANNOTATION_TYPE })
 @Retention(RetentionPolicy.RUNTIME)
 @MessageMapping
 @Documented
+@Repeatable(RabbitListeners.class)
```

Comment:
Oh! Yes, that's true. It doesn't matter where it is used, because Java just ignores (and warns) unknown annotations:
```
 warning: Cannot find annotation method 'value()' in type 'SuppressWarnings': class file for edu.umd.cs.findbugs.annotations.SuppressWarnings not found
```

It's only important that we always compile it with those classes on place.